scipy-1.0/paper.tex

```diff
@@ -281,6 +281,29 @@ \subsection*{Data structures}
 \input{ckdtree}
 \textbf{Sparse matrices}
 
+\texttt{scipy.sparse} offers seven sparse matrix data structures,
+also known as sparse formats. The most important ones are the row- 
+and column-compressed formats (CSR and CSC, respectively). 
+These offer fast major-axis indexing and fast matrix-vector multiplication,
+and are used heavily in scikit-learn.
+
+Over the last three years our sparse matrix handling internals have
+been rewritten and performance has been improved. Iterating over 
+and slicing of CSC and CSR matrices is now faster by up to 35\%,
+and the CSR to COO matrix format conversion is now several times faster.
+Importantly, SuperLU\cite{superlu_ug99} was updated to version 5.2.1, enhancing
+the low-level implementations leveraged by a subset of our \texttt{sparse}
+offerings.
+
+From a new features standpoint, \texttt{scipy.sparse} matrices and linear
+operators now support the Python matrix multiplication (@) operator when
+available. We've added \texttt{scipy.sparse.norm} and \texttt{scipy.sparse.random} 
+for computing sparse matrix norms and drawing random variates from arbitrary 
+distributions, respectively. Also, we've made a concerted effort to bring the
+\texttt{scipy.sparse} API into line with the equivalent NumPy API where possible,
+typically by adding support for kwargs like \texttt{axis=None} or
```

Comment:
The `copy=False` semantics are quite hard to understand (when is/isn't data being shared), and I'm not sure explicit examples contribute much here.  Perhaps enough to state that the API has been brought in line, without any examples?